modules/reference/pages/public-metrics-reference.adoc

```diff
@@ -908,6 +895,187 @@ The number of transform processors in a specific state (running, inactive, error
 
 ---
 
+=== Cloud storage metrics
+
+This section provides reference descriptions of the metrics available when Redpanda's xref:manage:tiered-storage.adoc[] is enabled.
+
+include::reference:partial$public_metrics_tip.adoc[]
+
+=== Prerequisites
+
+Cloud storage metrics are only enabled if you have:
+
+- xref:manage:tiered-storage.adoc[] enabled
+- The cluster property xref:reference:properties/cluster-properties.adoc#cloud_storage_enabled[cloud_storage_enabled] is set to `true`
+
+=== redpanda_cloud_storage_cache_space_size_bytes
+
+Sum of size of cached objects
```

Comment:
It looks like the rest of our metrics descriptions currently end with periods, should we standardize here as well?

Reply:
done